CONTRIBUTING.md

```diff
@@ -233,3 +235,55 @@ _NOTE: with all of these steps, if you have SSH Keys set up, you will want to us
 5. Make a new branch: `git checkout -b your_branch_name`.
 6. Pull in content from you private repo: `git pull private main`.
 7. Push your new work up to GitHub: `git push origin your_branch_name`.
+
+## Troubleshooting
+
+This section attempts to capture some of the more common troubleshooting techniques to try
+if you face issues in contributing.
+
+### General Install, compile or build issues
+
+Many of the common errors you'll face can be resolve by the following:
+
+1. Ensure you are using `Yarn` not `NPM`
```

Comment:
```suggestion
1. Ensure you are using `yarn` not `npm`
```

Maybe this is a personal choice, but generally I like referring to these as you'd type them in your terminal. Feel free to ignore this if you don't like this suggestion.